lib/common.sh

```diff
@@ -497,8 +497,10 @@ iterate(){
   local RUNS=0
   local COMMAND="$*"
   local TMP_RET TMP_RET_CODE
+  set +e
   TMP_RET="$(${COMMAND})"
   TMP_RET_CODE="$?"
+  set -e
 
   until [[ "${TMP_RET_CODE}" = 0 ]] || [[ "${SKIP_RETRIES}" = true ]]
   do
```

Comment:
```suggestion
  until "${COMMAND}" || [[ "${SKIP_RETRIES}" = true ]]; do
```
and deleting all TMP_RET stuff would be much cleaner.